misc/docker-ci/Dockerfile

```diff
@@ -38,13 +38,9 @@ RUN (cd openssl-${OPENSSL_VERSION} && \
 	make -j $(nproc) && make -j install_sw install_ssldirs)
 
 # cpan modules
-RUN apt-get install --yes cpanminus
-RUN apt-get install --yes libfcgi-perl libfcgi-procmanager-perl libipc-signal-perl libjson-perl liblist-moreutils-perl libplack-perl libscope-guard-perl libtest-exception-perl libwww-perl libio-socket-ssl-perl
-ENV PERL_CPANM_OPT="--mirror https://cpan.metacpan.org/"
-RUN sudo cpanm --notest Starlet Protocol::HTTP2
+RUN curl -L https://cpanmin.us | perl - --nootest --self-upgrade --sudo
+RUN cpanm --sudo --notest .
```

Comment:
I would appreciate it if you could adjust the comment above to say something like "install cpan modules listed in ./cpanfile" or something, as we recommend developers to look into Dockerfile to figure out the test dependencies.

It would be very scary for people to run `cpanm` without knowing what it does (I mean, `sudo cpanm --notest Starlet Protocol::HTTP` has the implication that it is installing the named modules, but `cpanm .` is very unclear).